packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-card/verification-status-banner.tsx

```diff
@@ -0,0 +1,200 @@
+                // poa is rejected,suspected, failed-resubmit
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--POA_POI`}>
+                            <Text size='xxxs' color={'white'}>
+                                <Localize i18n_default_text='Check your proof of address' />
+                            </Text>
+                        </div>
+                    </div>
+                );
+            } else if (need_poi_for_vanuatu && poa_acknowledged) {
+                return (
+                    <div className={`${card_classname}__footer--none`}>
+                        <Text as='p' size='xxs' align='center' color={'prominent'}>
+                            <Localize i18n_default_text='Proof of identity required' />
+                        </Text>
+                    </div>
+                );
+            } else if (need_poi_for_vanuatu && need_poa_submission) {
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--POA_POI`}>
+                            <Text size='xxxs' color={'white'}>
+                                <Localize i18n_default_text='Check your proof of identity and address' />
+                            </Text>
+                        </div>
+                    </div>
+                );
+            }
+        } else if (type_of_card === 'bvi' || type_of_card === 'labuan' || type_of_card === 'maltainvest') {
+            if (poi_poa_verified_for_bvi_labuan_maltainvest) {
+                return null; //both verified-no banner
+            } else if (poi_acknowledged_for_bvi_labuan_maltainvest && poa_acknowledged) {
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--pending`}>
+                            <Text size='xxxs' color={'prominent'}>
+                                <Localize i18n_default_text='Pending verification' />
+                            </Text>
+                        </div>
+                    </div>
+                );
+            } else if (need_poa_resubmission && poi_acknowledged_for_bvi_labuan_maltainvest) {
+                // poa is rejected,suspected, failed-resubmit
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--POA_POI`}>
+                            <Text size='xxxs' color={'white'}>
+                                <Localize i18n_default_text='Check your proof of address' />
+                            </Text>
+                        </div>
+                    </div>
```

Comment:
This is repeated everywhere in this component. Make it a component that accepts a children. Something like this:
```
<VerificationStatusWrapper>
    <Localize i18n_default_text='Check your proof of address' />
</VerificationStatusWrapper>
```
This way you have way less repetition